requirements.txt

```diff
@@ -5,4 +5,5 @@ distributed==2021.4.1
 pandas>=1.0,<1.3.0dev0
 numba>=0.53.1
 tdqm
+tensorflow-metadata
```

Comment:
This is another place where I wonder if it's worth adding the dependency, or if we should implement something similar ourselves.

Reply:
So for consistency I think it would be best to make use of the protobuf schemas defined in `tensorflow-metadata`, this would also open up the possibility to offer deep integration with TFX in a separate library. We could, however, choose to copy over the relevant protobuf schemas to our repo. The reason I didn't do that now is because I am not sure if that's possible with the license they use. The `tensorflow-metadata` dependency is just a couple of protobuf definitions so it is super lightweight, so I think it could be fine.